app/models/user.rb

```diff
@@ -4,7 +4,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :first_name, :last_name, presence: true, allow_blank: false
-
+  validates :first_name, :last_name, presence: true
```

Comment:
I had removed it from your sublime; I'm not completely sure if you wanted to remove it in the end; sorry @ClementLaussucq  ;-(